basket/news/tasks.py

```diff
@@ -241,6 +245,10 @@ def fxa_verified(data):
     # used to be handled by the fxa_register view
     email = data['email']
     fxa_id = data['uid']
+    create_date = data.get('createDate')
+    if create_date:
+        create_date = gmtime(create_date)
```

Comment:
`gmtime()` returns a `struct_time` tuple, not a datetime object. You want [datetime.datetime.utcfromtimestamp](https://docs.python.org/2/library/datetime.html#datetime.datetime.utcfromtimestamp)